content/zh-cn/docs/reference/kubernetes-api/cluster-resources/event-v1.md

```diff
@@ -0,0 +1,935 @@
+-->
+Event 是集群中某个事件的报告。它一般表示系统的某些状态变化。
+Event 的保留时间有限，触发器和消息可能会随着时间的推移而演变。
+事件消费者不应假定给定原因的事件的时间所反映的是一致的下层触发因素，或具有该原因的事件的持续存在。
+Events 应被视为通知性质的、尽最大努力而提供的补充数据。
+
+<hr>
+
+- **apiVersion**: events.k8s.io/v1
+
+- **kind**: Event
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准的对象元数据。 更多信息: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+<!--
+- **eventTime** (MicroTime), required
+
+  eventTime is the time when this Event was first observed. It is required.
+-->
+
+- **eventTime** (MicroTime)，必需
+
+  evenTime 是该事件首次被观察到的时间。它是必需的。
+
+  <a name="MicroTime"></a>
+
+  <!--
+  *MicroTime is version of Time with microsecond level precision.*
+  -->
+  
+  **MicroTime 是微秒级精度的 Time 版本**
+
+- **action** (string)
+
+  <!--
+  action is what action was taken/failed regarding to the regarding object. It is machine-readable. This field cannot be empty for new Events and it can have at most 128 characters.
+  -->
+  action 是针对相关对象所采取的或已失败的动作。字段值是机器可读的。对于新的 Event，此字段不能为空，
+  且最多为 128 个字符。
+
+- **deprecatedCount** (int32)
+
+  <!--
+  deprecatedCount is the deprecated field assuring backward compatibility with core.v1 Event type.
+  -->
+  deprecatedCount 是确保与 core.v1 Event 类型向后兼容的已弃用字段。
+
+- **deprecatedFirstTimestamp** (Time)
+
+  <!--
+  deprecatedFirstTimestamp is the deprecated field assuring backward compatibility with core.v1 Event type.
+  -->
+  deprecatedFirstTimestamp 是确保与 core.v1 Event 类型向后兼容的已弃用字段。
+
+  <a name="Time"></a>
+  
+  <!--
+  *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+  -->
+  **Time 是对 time.Time 的封装。Time 支持对 YAML 和 JSON 进行正确封包。为 time 包的许多函数方法提供了封装器。**
+
+- **deprecatedLastTimestamp** (Time)
+
+  <!--
+  deprecatedLastTimestamp is the deprecated field assuring backward compatibility with core.v1 Event type.
+  -->
+  deprecatedLastTimestamp 是确保与 core.v1 Event 类型向后兼容的已弃用字段。
+
+  <a name="Time"></a>
+
+  <!--
+  *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+  --> 
+  **Time 是对 time.Time 的封装。Time 支持对 YAML 和 JSON 进行正确封包。为 time 包的许多函数方法提供了封装器。**
+
+- **deprecatedSource** (EventSource)
+
+  <!--
+  deprecatedSource is the deprecated field assuring backward compatibility with core.v1 Event type.
+  -->  
+  deprecatedSource 是确保与 core.v1 Event 类型向后兼容的已弃用字段。
+
+  <a name="EventSource"></a>
+  
+  <!--
+  *EventSource contains information for an event.*
+  -->
+  
+  **EventSource 包含事件信息。**
+
+  - **deprecatedSource.component** (string)
+
+    <!--
+    Component from which the event is generated.
+    -->
+    生成事件的组件。
```

Comment:
The parse result is different with English page.
@tengqm do you know how to fix it?